c-dependencies/js-compute-runtime/js-compute-builtins.h

```diff
@@ -81,7 +81,28 @@ bool hasWizeningFinished();
 bool isWizening();
 void markWizeningAsFinished();
 
-bool define_fastly_sys(JSContext *cx, JS::HandleObject global);
+class FastlyOptions {
+private:
+  uint8_t mask = 0;
```

Comment:
I would probably make booleans for each option instead of masking. We won't make many of these values, so we probably don't need to resort to bit-packing here :)